app/src/main/java/com/android/ootd/ui/post/items/AddItemsViewModel.kt

```diff
@@ -263,6 +269,27 @@ open class AddItemsViewModel(
                 _addOnSuccess.value = false
                 return@launch
               }
+
+              // Check if we need to schedule background upload (if offline, we got a local URI)
+              val uri = Uri.parse(uploaded.imageUrl)
```

Comment:
Instead of using `Uri.parse()` you can use this 

```suggestion
              val uri = uploaded.imageUrl.toUri()
```

This is also what AndroidStudio is suggesting when I looked your code.

Reply:
That's the problem of coding in vscode, it doesn't show me these tips 😢 

thanks